com.unity.visualeffectgraph/Documentation~/VisualEffectGraphWindow.md

```diff
@@ -16,34 +16,35 @@ To open the Visual Effect Graph window, you can use any of the following methods
*: Opens the Visual Effect Graph [manual](https://docs.unity3d.com/Packages/com.unity.visualeffectgraph@latest/).<br/><br/>**Dropdown**:<br/><br/>**VFX Graph Additions**: Installs pre-made visual effects and utility operators made with Visual Effect Graph. These include Bonfire, Lightning, Smoke and Sparks effect examples. There are also various operators and Subgraph Blocks:<br/><br/>&#8226; **Get Strip Progress**: A subgraph that calculates the progress of a particle in the strip in the range 0 to 1. You can use this to sample curves and gradients to modify the strip based on its progress.<br/><br/>&#8226; **Encompass (Point)**: A subgraph that grows the bounds of an AABox to encompass a point.<br/><br/>&#8226; **Degrees to Radians and Radians to Degrees**: Subgraphs that help you to convert between radians and degrees within your graph.<br/><br/>**Output Event Helpers**: This version of the Visual Effect Graph introduces new helper scripts to the OutputEvent Helpers sample to help you to set up OutputEvents:<br/><br/>&#8226; **Cinemachine Camera Shake**: An Output Event Handler script that triggers a Camera shake through a [Cinemachine Impulse Source](https://docs.unity3d.com/Packages/com.unity.cinemachine@latest/index.html?subfolder=/manual/CinemachineImpulseSourceOverview.html), on a given output event.<br/><br/>&#8226; **Play audio**:  An Output Event Handler script that plays a single AudioSource on a given output event.<br/><br/>&#8226; **Spawn a Prefab**:  An Output Event Handler script that spawns Prefabs (managed from a pool) on a given output event. It uses position, angle, scale, and lifetime to position the Prefab and disable it after a delay. To synchronize other values, you can use other scripts inside the Prefab:<br/><br/>&#8226; **Change Prefab Light**: An example that demonstrates how to synchronize a light with your effect.<br/><br/>&#8226; **Change Prefab RigidBody Velocity**: An example that demonstrates how to synchronize changing the velocity of a RigidBody with your effect.<br/><br/>&#8226; **RigidBody**: An Output Event Handler script that applies a force or velocity change to a RigidBody on a given output event.<br/><br/>&#8226; **Unity Event**: An Output Event Handler that raises a UnityEvent on a given output event.<br/><br/>&#8226; **[VFX Graph Home](https://unity.com/visual-effect-graph)**: Opens the Visual Effect Graph home page.<br/><br/>&#8226; **[Forum](https://forum.unity.com/forums/visual-effect-graph.428/)**: Open a Visual Effect Graph sub-forum.<br/><br/>&#8226; **[Github] [Spaceship Demo](https://github.com/Unity-Technologies/SpaceshipDemo)**: Opens a repository of AAA Playable First person demo showcasing effects made with Visual Effect Graph and rendered with the High Definition Render Pipeline.<br/><br/>&#8226; **[Github] [VFX Graph Samples](https://github.com/Unity-Technologies/VisualEffectGraph-Samples)**: Opens a repository that contains sample scenes and visual effects made with Visual Effect Graph.|
```

Comment:
'Open a Visual Effect Graph sub-forum'. 'Opens the Visual Effect Graph home page'. Pardon my ignorance, just a question, is this intended that sometimes we say open and sometimes opens. Aren't these supposed to be consistent?

Reply:
Fixed